runtime/lua/vim/lsp/rpc.lua

```diff
@@ -325,7 +323,14 @@ local function start(cmd, cmd_args, dispatchers, extra_spawn_params)
     end
     handle, pid = uv.spawn(cmd, spawn_params, onexit)
     if handle == nil then
-      error(string.format("start `%s` failed: %s", cmd, pid))
+      local msg = string.format("Spawning language server with cmd: `%s` failed", cmd)
```

Comment:
```suggestion
      local msg = string.format("Starting language server with cmd: `%s` failed", cmd)
```

Reply:
Seems more clear for not that technical users (and not gamers :D). No big deal feel free to ignore.

Reply:
I called it "spawning" to draw attention to uv_spawn (which is what failed). I actually had it as starting in the first version of this PR :P